app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+from app.players.dwarves.dwarf import Dwarf
+from app.players.elves.elf import Elf
+from app.players.player import Player
+
+
+def calculate_team_total_rating(players: list[Player]) -> int:
+    total = 0
+
+    for player in players:
+        total += player.get_rating()
+
+    return total
+
+
+def elves_concert(elves: list[Elf]) -> None:
+    for elf in elves:
+        elf.play_elf_song()
+
+
+def feast_of_the_dwarves(dwarves: list[Dwarf]) -> None:
+    for dwarf in dwarves:
```

Comment:
Check prev comment

Reply:
same as previous

Reply:
basically if we use list comprehension here we will create a list full of None values that we will never use, so for loop is just fine :)